src/SfxWeb/src/app/modules/apptypes-viewer/apptype-viewer/apptype-viewer.component.html

```diff
@@ -1,39 +1,37 @@
-<div class="tile-container" data-cy="tiles">
-  <app-dashboard-text-tile [barClass]="'badge-upgrade-active'" [count]="usage.activeAppTypes.length" *ngIf="usage"
```

Comment:
did the tiles get fully removed or just moved?

Reply:
it has been moved to apptypes.component.html